### PR TITLE
mapped a lot of net/minecraft/client/screen/inventory

### PR DESCRIPTION
**Pull request by justliliandev**
mapped also some other stuff I tumbled into

### CODE REVIEW

File: data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping [hunk 1]
```diff
@@ -136,3 +134,29 @@ CLASS net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen
 			ARG 1 stack
 		METHOD setChanged ()V
 			COMMENT Called when the stack in a Slot changes
+	CLASS ItemPickerMenu
+		FIELD items Lnet/minecraft/core/NonNullList;
+			COMMENT the list of items in this container
```
Comment (Darkhax) [inline data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping]: This should start with an upper case and end with a period.

File: data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping [hunk 2]
```diff
@@ -136,3 +134,29 @@ CLASS net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen
...
+		METHOD <init> (Lnet/minecraft/world/entity/player/Player;)V
+			ARG 1 player
+		METHOD canDragTo (Lnet/minecraft/world/inventory/Slot;)Z
+			COMMENT Returns true if the player can "drag-spilt" items into this slot,. returns true by default. Called to check if the slot can be added to a list of Slots to split the held ItemStack across.
```
Comment (Darkhax) [inline data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping]: The `,` is not needed. The second returns should also be capitalized.

File: data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping [hunk 3]
```diff
@@ -136,3 +134,29 @@ CLASS net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen
...
+		METHOD canTakeItemForPickAll (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/inventory/Slot;)Z
+			COMMENT Called to determine if the current slot is valid for the stack merging (double-click) code. The stack passed in is null for the initial slot that was double-clicked.
+			ARG 1 stack
+			ARG 2 slot
+		METHOD quickMoveStack (Lnet/minecraft/world/entity/player/Player;I)Lnet/minecraft/world/item/ItemStack;
+			COMMENT Handle when the stack in slot {@code index} is shift-clicked. Normally this moves the stack between the player inventory and the other inventory(s).
+			ARG 1 player
+			ARG 2 index
+		METHOD scrollTo (F)V
+			COMMENT Updates the gui slots ItemStack's based on scroll position.
```
Comment (diesieben07) [inline data/net/minecraft/client/gui/screens/inventory/CreativeModeInventoryScreen.mapping]: Should be `slot's ItemStacks`

File: data/net/minecraft/world/inventory/AbstractContainerMenu.mapping [hunk 4]
```diff
@@ -1,4 +1,7 @@
 CLASS net/minecraft/world/inventory/AbstractContainerMenu
+	METHOD <init> (Lnet/minecraft/world/inventory/MenuType;I)V
+		ARG 1 menuType
+		ARG 2 containerID
```
Comment (diesieben07) [inline data/net/minecraft/world/inventory/AbstractContainerMenu.mapping]: The field is called `containerId`, we should mimic that.

**Review by diesieben07 — CHANGES_REQUESTED** [review]

**Review by diesieben07 — DISMISSED** [review]

**Review by Darkhax — DISMISSED** [review]

**Review by jriwanek — DISMISSED** [review]

File: data/net/minecraft/client/gui/GuiComponent.mapping [hunk 5]
```diff
@@ -99,6 +99,15 @@ CLASS net/minecraft/client/gui/GuiComponent
 		ARG 5 y2
 		ARG 6 colorFrom
 		ARG 7 colorTo
+	METHOD fillGradient (Lcom/mojang/blaze3d/vertex/PoseStack;IIIIIII)V
+		ARG 0 poseStack
```
Comment (Lanse505) [inline data/net/minecraft/client/gui/GuiComponent.mapping:113]: Correct, but please update the rest of the class to fix this as well.
There are still a lot of "matrixPos" entries in there that needs to be renamed.

File: data/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.mapping [hunk 6]
```diff
@@ -72,8 +76,8 @@ CLASS net/minecraft/client/gui/screens/inventory/AbstractContainerScreen
 	METHOD renderBg (Lcom/mojang/blaze3d/vertex/PoseStack;FII)V
 		ARG 1 matrixStack
```
Comment (Lanse505) [inline data/net/minecraft/client/gui/screens/inventory/AbstractContainerScreen.mapping]: ``matrixStack`` -> ``poseStack``

Apply this change to any other instances you find.

File: data/net/minecraft/world/entity/player/Player.mapping [hunk 7]
```diff
@@ -190,12 +190,14 @@ CLASS net/minecraft/world/entity/player/Player
...
+	METHOD openJigsawBlock (Lnet/minecraft/world/level/block/entity/JigsawBlockEntity;)V
+		ARG 1 jigsawBlock
 	METHOD openMinecartCommandBlock (Lnet/minecraft/world/level/BaseCommandBlock;)V
 		ARG 1 commandBlock
 	METHOD openStructureBlock (Lnet/minecraft/world/level/block/entity/StructureBlockEntity;)V
-		ARG 1 structure
+		ARG 1 structureBlock
 	METHOD openTextEdit (Lnet/minecraft/world/level/block/entity/SignBlockEntity;)V
-		ARG 1 signTile
+		ARG 1 signBlock
```
Comment (Lanse505) [inline data/net/minecraft/world/entity/player/Player.mapping]: ``signBlock`` -> ``signEntity``

File: data/net/minecraft/world/entity/player/Player.mapping [hunk 8]
```diff
@@ -190,12 +190,14 @@ CLASS net/minecraft/world/entity/player/Player
 	METHOD openItemGui (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/InteractionHand;)V
 		ARG 1 stack
 		ARG 2 hand
+	METHOD openJigsawBlock (Lnet/minecraft/world/level/block/entity/JigsawBlockEntity;)V
+		ARG 1 jigsawBlock
 	METHOD openMinecartCommandBlock (Lnet/minecraft/world/level/BaseCommandBlock;)V
 		ARG 1 commandBlock
 	METHOD openStructureBlock (Lnet/minecraft/world/level/block/entity/StructureBlockEntity;)V
-		ARG 1 structure
+		ARG 1 structureBlock
```
Comment (Lanse505) [inline data/net/minecraft/world/entity/player/Player.mapping]: ``structureBlock`` -> ``structureEntity``

File: data/net/minecraft/world/entity/player/Player.mapping [hunk 9]
```diff
@@ -190,12 +190,14 @@ CLASS net/minecraft/world/entity/player/Player
 	METHOD openItemGui (Lnet/minecraft/world/item/ItemStack;Lnet/minecraft/world/InteractionHand;)V
 		ARG 1 stack
 		ARG 2 hand
+	METHOD openJigsawBlock (Lnet/minecraft/world/level/block/entity/JigsawBlockEntity;)V
+		ARG 1 jigsawBlock
```
Comment (Lanse505) [inline data/net/minecraft/world/entity/player/Player.mapping]: ``jigsawBlock`` -> ``jigsawEntity``

File: data/net/minecraft/world/inventory/tooltip/BundleTooltip.mapping [hunk 10]
```diff
@@ -0,0 +1,4 @@
+CLASS net/minecraft/world/inventory/tooltip/BundleTooltip
+	METHOD <init> (Lnet/minecraft/core/NonNullList;I)V
+		ARG 1 itemStacks
```
Comment (Lanse505) [inline data/net/minecraft/world/inventory/tooltip/BundleTooltip.mapping]: ``itemStacks`` -> ``items``

**Review by Lanse505 — CHANGES_REQUESTED** [review]

**Review by Lanse505 — APPROVED** [review]

**Review by jriwanek — APPROVED** [review]

**Review by Darkhax — APPROVED** [review]

**Review by marchermans — APPROVED** [review]

**Pull request merged**